data-prepper-plugins/otel-trace-source/README.md

```diff
@@ -7,7 +7,7 @@ This is a source which follows the [OTLP Protocol](https://github.com/open-telem
 Example `.yaml` configuration:
 ```
```

Comment:
I notice you marked a lot of code blocks as `yaml`. You may be interested in declaring the code block as `yaml` if you do another revision.